docs/stories/components/DatePicker.stories.js

```diff
@@ -28,7 +28,7 @@ import moment from 'moment';
 import { DatePicker } from '@buffetjs/core';
 
 function Example() {
-  const [value, setValue] = useState(moment());
+  const [value, setState] = useState(moment([2020, 0, 11]));
```

Comment:
If you change the name here you also need to change in the callback of the example.